### PR TITLE
eslint test/unit_tests folder as well

### PR DESCRIPTION
**Pull request by lipis**

### CODE REVIEW

File: .eslintrc.yaml [hunk 1]
```diff
@@ -10,13 +11,19 @@ globals:
...
+  entities: true
   ko: true
   LRUCache: true
+  moment: true
+  notification_service: true
   pako: true
   Raygun: true
+  search_service: true
+  user_service: true
+  web_socket_service: true
```
Comment (lipis) [inline .eslintrc.yaml]: if there are better ideas.. I'm all ears :)

**Comment by herrmannplatz** [comment]
const > let

**Comment by ffflorian** [comment]
Why do you add `const self = this` everywhere? You could / should use `this`.

**Comment by lipis** [comment]
It was the automatic `eslint --fix` but since there were too many errors again further in travis.. will split this into two again.. only the `unit_tests` for this PR

**Comment by ffflorian** [comment]
Okay. If no further changes will be added, I can approve.

**Pull request merged**